package.json

```diff
@@ -35,19 +41,33 @@
     "karma-mocha-reporter": "2.2.5",
     "karma-requirejs": "1.1.0",
     "karma-safari-launcher": "1.0.0",
-    "puppeteer": "5.4.0",
+    "load-grunt-tasks": "^5.1.0",
+    "postcss": "^8.1.4",
+    "postcss-color-converter": "^1.0.1",
+    "postcss-remove-prefixes": "^1.2.0",
+    "postcss-unprefix": "^2.1.4",
+    "puppeteer": "5.3.1",
     "requirejs": "2.3.6",
     "selenium-standalone": "6.20.1",
     "selenium-webdriver": "3.6.0",
     "string.prototype.endswith": "1.0.0",
     "string.prototype.startswith": "1.0.0",
+    "stylelint": "^13.7.2",
+    "stylelint-config-recommended": "^3.0.0",
+    "stylelint-config-standard": "^20.0.0",
+    "stylelint-order": "^4.1.0",
     "wdio-chromedriver-service": "^6.0.4",
     "webdriverio": "^6.6.0"
   },
+  "browserslist": [
+    "defaults"
```

Comment:
This is used by autoprefixer to know what browsers to cater for. I just used the default setting.